discovery-provider/scripts/start.sh

```diff
@@ -85,18 +103,16 @@ if [[ "$audius_discprov_dev_mode" == "true" ]]; then
         echo "Finished running migrations"
     fi
 
-    ./scripts/dev-server.sh 2>&1 | tee >(logger -t server) server.log &
+    ./scripts/dev-server.sh 2>&1 | tee >(logger -t server) & # docker exec -it <disc-prov container> tail -f /var/log/discprov-server.log
```

Comment:
small nit - can you put these example commands above the actual code so the line isn't too long? eg 

```suggestion
    # filter tail to server logs with `docker exec -it <disc-prov container> tail -f /var/log/discprov-server.log`
    ./scripts/dev-server.sh 2>&1 | tee >(logger -t server) & 
```